Utilities/Thread.cpp

```diff
@@ -3029,19 +3029,26 @@ void thread_ctrl::set_native_priority(int priority)
 		sig_log.error("SetThreadPriority() failed: %s", fmt::win_error{GetLastError(), nullptr});
 	}
 #else
-	int policy;
-	struct sched_param param;
-
-	pthread_getschedparam(pthread_self(), &policy, &param);
-
+	// available niceness for nonroot: 0~19
+	int linuxprio = 0;
```

Comment:
What about the other platforms like BSD and apple?

Reply:
oops. fixed again.